lua/ulx/modules/sh/freeze.lua

```diff
@@ -0,0 +1,50 @@
+CATEGORY_NAME = "CFC"
+
+do
```

Comment:
why do we need the `do end`?

Reply:
i like having locals kept in a nice box, but i guess it would be kinda unnecessary.